src/transformers/modeling_utils.py

```diff
@@ -978,36 +978,38 @@ def generate(
 
         Examples::
 
+            from transformers import AutoTokenizer, AutoModelForCausalLM
```

Comment:
Same here

Reply:
Can't manage to have a clean doctest with such a big example